test/engine/update.test.lua

```diff
@@ -160,3 +160,110 @@ t:update({{'+', '[1]', 50}})
 s:update({1}, {{'=', 'field3', {d = 30, e = 31, f = 32}}})
 
 s:drop()
+
+--
+-- gh-3378: allow update absent nullable fields
+
+-- '!'
+s = box.schema.create_space('test')
+pk = s:create_index('pk')
+s:replace{1, 2}
+s:update({1}, {{'!', 4, 0}})
+s:drop()
```

Comment:
You don't have to drop space, just use another tuple or delete it - it should be a bit faster.